framework/widgets/Pjax.php

```diff
@@ -171,6 +171,11 @@ public function run()
         $response->setStatusCode(200);
         $response->format = Response::FORMAT_HTML;
         $response->content = $content;
+        
+        if (!isset($response->headers['X-Pjax-Url'])) {
```

Comment:
you can use http://www.yiiframework.com/doc-2.0/yii-web-headercollection.html#setDefault()-detail and make it one line only